graders/c/requirements.txt

```diff
@@ -0,0 +1,19 @@
+asn1crypto==0.24.0
+cryptography==2.1.4
+cycler==0.10.0
+idna==2.6
+keyring==10.6.0
+keyrings.alt==3.0
+kiwisolver==1.3.1
+matplotlib==3.3.4
+mpmath==1.2.1
+numpy==1.20.1
+Pillow==8.1.0
+pycrypto==2.6.1
+pygobject==3.26.1
+pyparsing==2.4.7
+python-dateutil==2.8.1
+pyxdg==0.25
+SecretStorage==2.3.1
+six==1.11.0
+sympy==1.7.1
```

Comment:
Are all of these actually required? This seems excessive at first glance.

Reply:
Good point. I just did a dump on the status of what was in the previous version. I'll trim it down a bit.